ni_measurementlink_service/_internal/parameter/serialization_strategy.py

```diff
@@ -107,12 +156,80 @@ def vector_decoder(field_index, key):
     return vector_decoder
 
 
+def _decode_varint(buffer, pos):
+    mask = (1 << 64) - 1
+    result_type = int
+    result = 0
+    shift = 0
+    while 1:
+        b = buffer[pos]
+        result |= (b & 0x7F) << shift
+        pos += 1
+        if not (b & 0x80):
+            result &= mask
+            result = result_type(result)
+            return (result, pos)
+        shift += 7
+        if shift >= 64:
+            raise Exception("Too many bytes when decoding varint.")
```

Comment:
This was a _DecodeError in the original code. Can we use something more descriptive than Exception, like ValueError?